pkg/cli/node.go

```diff
@@ -64,9 +66,10 @@ func runLsNodes(cmd *cobra.Command, args []string) error {
 	_, rows, err := runQuery(
 		conn,
 		makeQuery(`SELECT node_id FROM crdb_internal.gossip_liveness
-               WHERE decommissioning = false OR split_part(expiration,',',1)::decimal > now()::decimal`),
+               WHERE membership = 'active' OR split_part(expiration,',',1)::decimal > now()::decimal`),
```

Comment:
Is the old column still around? I.e. will `./cockroach-20.1 node ls` work against 20.2 cluster? I think the answer is yes, just checking.